go/integration_test/processor/scenario.go

```diff
@@ -222,7 +222,7 @@ func (c *scenarioContext) SetTransientStorage(tosca.Address, tosca.Key, tosca.Wo
 }
 
 func (c *scenarioContext) AccessAccount(tosca.Address) tosca.AccessStatus {
-	panic("implement me")
+	return false // correct value only important inside of the interpreter
```

Comment:
sorry, I don't really understand what this comment means.

Reply:
should this not check in some list of accessed accounts held by the `scenarioContext` ?

Reply:
I added an access list to the `scenarioContext`, I thought it should be part of the WorldState but the functions only get one address as an argument.